.github/CODEOWNERS

```diff
@@ -1,11 +1,3 @@
-modules/apps/foundation/configuration-admin/*    @jorgeferrer
-modules/apps/foundation/contacts/*    @jorgeferrer
-modules/apps/foundation/password-policies-admin/*    @jorgeferrer
-modules/apps/foundation/plugins-admin/*    @jorgeferrer
-modules/apps/foundation/portal-configuration/*    @jorgeferrer
-modules/apps/foundation/portal-instances/*    @jorgeferrer
-modules/apps/foundation/portal-settings/*    @jorgeferrer
-modules/apps/foundation/roles/*    @jorgeferrer
-modules/apps/foundation/server/*    @jorgeferrer
-modules/apps/foundation/user-groups-admin/*    @jorgeferrer
-modules/apps/foundation/users-admin/*    @jorgeferrer
\ No newline at end of file
+modules/apps/frontend-data-set/*    @liferay-frontend
+modules/apps/frontend-editor/*    @liferay-frontend
+modules/apps/frontend-taglib/*    @liferay-frontend
```

Comment:
That's a great beginning @herrardo . We can add some more:

- most of  `modules/apps/frontend-js`  (some exceptions apply)
- `modules/apps/cookies/cookies-banner-web`

Additionally, we may need to clarify:

- Some stuff in `modules/apps/frontend-taglib/*` (dynamic-section, form-navigation) would not belong to @liferay-frontend 

That's what I can tell from the top of my head, to have a more or less accurate representation